rocAL/rocAL_pybind/amd/rali/plugin/pytorch.py

```diff
@@ -22,6 +22,7 @@ def next(self):
 
     def __next__(self):
         if b.getRemainingImages(self.loader._handle) < self.bs:
+            b.raliResetLoaders(self.loader._handle)
```

Comment:
this looks like a hack to fix the issue. The raliResetLoaders() has to be called in a seperate function which can be called at the end of every epoch. I think the Pytorch training code has already this routine and we can use similar code

Reply:
Please call the reset() function instead